module/armor/fan_armor.hpp

```diff
@@ -1,7 +1,31 @@
 #pragma once
 
+#include <vector>
+
 #include "abstract_armor.hpp"
+#include "module/buff/abstract_object.hpp"
```

Comment:
Additional relative path specifying in not need:

```suggestion
#include "abstract_object.hpp"
```

Reply:
Agree